newupdater/__init__.py

```
@@ -1,3 +1,4 @@
 # -*- coding: utf-8 -*-
 """
 """
+from .new_updater import *
```

Ruff (0.13.3)

`from .new_updater import *` used; unable to detect undefined names

(F403)